src/styles/_common/_colors.scss

```diff
@@ -111,8 +111,14 @@ $palette: (
     'border': rgba(0, 60, 88, 0.3),
     'border-active': rgba(0, 60, 88, 0.7),
   ),
-  cookieAlert: (
+  'cookieAlert': (
```

Comment:
@asantos00 cookieAlert was already camelCased but I agree that dashed should be standard.